sdk/sdktypes/consts.go

```diff
@@ -0,0 +1,6 @@
+package sdktypes
+
+const (
+	SchedulerEventTriggerType = "scheduler"
```

Comment:
why not put this in `trigger.go`?

Reply:
because it's related to both event, trigger and dispatcher. So I created a new file with consts